doc/user/layouts/shortcodes/cluster-options.html

```diff
@@ -0,0 +1,9 @@
+Field                               | Value      | Description
+------------------------------------|------------|-------------------------------------
+`MANAGED`                           | `bool`     | Is the cluster a managed cluster?
+`SIZE`                              | `text`     | The size of the cluster replicas. For valid sizes, see [cluster replica sizes](/sql/create-cluster-replica#sizes).
+`REPLICATION FACTOR`                | `text`     | The replication factor of the cluster. Can be 0 to disable computations.
+`AVAILABILITY ZONES`                | `text[]`   | If you want the cluster replicas to reside in specific availability zones. You must specify a list of [AWS availability zone IDs] in either `us-east-1` or `eu-west-1`, e.g. `use1-az1`. Note that we expect the zone's ID, rather than its name.
```

Comment:
@antiguru, where did you want `AWS availability zone IDs` to point to? That link is currently broken.

Reply:
Ah, that's a link that's on the `CREATE CLUSTER` and `CREATE CLUSTER REPLICA` pages:

https://github.com/MaterializeInc/materialize/blob/bc371e6a20cc694fd38096c188058d9c74a892e6/doc/user/content/sql/create-cluster-replica.md#L115

Reply:
Thanks, Nikhil. Fixed.